rpc_sidecar/src/node_client.rs

```diff
@@ -323,8 +554,8 @@ impl Error {
                 | ErrorCode::InvalidTransactionUnableToCalculateGasCost
                 | ErrorCode::InvalidTransactionPricingMode
                 | ErrorCode::InvalidTransactionUnspecified
-                | ErrorCode::InvalidTransactionOrDeployUnspecified,
-            ) => Self::InvalidTransaction, // TODO: map transaction errors to proper variants
+                | ErrorCode::InvalidTransactionOrDeployUnspecified),
+            ) => Self::InvalidTransaction(InvalidTransactionOrDeploy::from(err)), // TODO: map transaction errors to proper variants
```

Comment:
```suggestion
            ) => Self::InvalidTransaction(InvalidTransactionOrDeploy::from(err)), // TODO: map transaction errors to proper variants
```
you can remove the todo now